provisioner/plain-v0/README.md

```diff
@@ -1,3 +1,197 @@
+InstallationSucceeded Phase if the application of resources to the cluster was successful.
+
+```
+NAME                 DESIRED BUNDLE   INSTALLED BUNDLE   INSTALL STATE           AGE
+my-bundle-instance   my-bundle        my-bundle          InstallationSucceeded   11s
+```
+
+### Make bundle content available but do not install it
+
+There is a natural separation between sourcing of the content and application of that content via two separate RukPak
+APIs, `Bundle` and `BundleInstance`. A user can specify a particular `Bundle` to be available in the cluster for
+inspection before any application of the resources. Given a `Bundle` resource named `my-bundle`, the plain-v0
+provisioner will pull down and unpack the bundle to a variety of storage backends, such as ConfigMaps or
+PersistentVolumes.
+
+By default, `rukpak-system` is the default namespace for deploying plain-v0 provisioner-related system resources.
+
+Surfacing the content of a bundle in a more user-friendly way, via a plugin or additional API, is on the RukPak roadmap.
+
+### Pivoting between bundle versions
+
+The `BundleInstance` API is meant to indicate the version of the bundle that should be active within the cluster. Given
+two unpacked bundles in the cluster,
+`my-bundle-v0.0.1` and `my-bundle-v0.0.2`, the `spec.bundleName` field of the related `BundleInstance` can be updated to
+pivot to a different version. For example by pivoting from an older version, to the newer version, by updating
+the `spec.bundleName` field the `plain-v0` provisioner will create the new bundle content on-cluster and remove the old
+content. The provisioner also continually reconciles the created content via dynamic watches to ensure that all
+resources referenced by the bundle are present on the cluster.
+
+## Running locally
+
+To experiment with the `plain-v0` provisioner locally, first setup a local cluster, or simply
+have [kind](https://kind.sigs.k8s.io/) installed locally.
+
+Once the cluster has been setup, take the following step from the root of the repository:
+
+* Run `make run` to build and deploy the provisioner onto the local cluster.
+
+From there, create some `Bundles` and `BundleInstance` types to see the provisioner in action. For an example bundle to
+use, the [combo operator](https://github.com/operator-framework/combo) is a good candidate.
+
+Create the bundle:
+
+```yaml
+apiVersion: core.rukpak.io/v1alpha1
+kind: Bundle
+metadata:
+  name: combo-0.0.1
+spec:
+  image: quay.io/tyslaton/combo-plain-bundle:v0.0.1
+  provisionerClassName: core.rukpak.io/plain-v0
+```
+
+Check the Bundle status via `kubectl get bundle combo-0.0.1`. Eventually the Bundle should show up as Unpacked.
+
+```
+NAME          IMAGE                                        PHASE      AGE
+combo-0.0.1   quay.io/tyslaton/combo-plain-bundle:v0.0.1   Unpacked   17h
+```
+
+Create the combo `BundleInstance` referencing the combo `Bundle` available in the cluster.
```

Comment:
follow-up: mention we can create a set of Bundles/BundleInstances at the same time (e.g. it works with gitops and doesn't have any resource ordering requirements like OLM).